.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,62 @@
+name: Bug Report
+description: Create a report to help us reproduce and correct the bug
+labels: ['bug', 'needs-triage']
+
+body:
+- type: markdown
+  attributes:
+    value: >
+      #### Before submitting a bug, please make sure the issue hasn't been already
+      addressed by searching through [the past issues](https://github.com/probabl-ai/skore/issues).
+- type: textarea
+  attributes:
+    label: Describe the bug
+    description: >
+      A clear and concise description of the bug.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Steps/Code to Reproduce
+    description: |
+      Please add a minimal code example that can reproduce the error when running it. Be as succinct as possible, **do not depend on external data files**.
+    placeholder: |
+      ```
+      Describe each step to reproduce the problem, along with related sample code if relevant.
+      ```
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Expected Behavior
+    description: >
+      A clear and concise description of what you expected to happen.
+    placeholder: >
+      Example: No error is thrown.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Actual Behavior
+    description: |
+      A clear and concise description of what actually happened. If you observe an error, please paste the error message including the **full traceback** of the exception.
```

Comment:
Can we help users format there traceback ? I'm not familiar with issue templates but I know that python traceback can be formatted using `pytb` as a language selector for markdown code blocks.

Reply:
I did it.